provider-blackduck/src/test/java/com/synopsys/integration/alert/provider/blackduck/processor/message/RuleViolationClearedNotificationMessageExtractorTest.java

```diff
@@ -209,6 +226,9 @@ private ProjectVersionComponentVersionView createProjectVersionComponentVersionV
         meta.setLinks(List.of(resourceLink));
         projectVersionComponentVersionView.setMeta(meta);
 
+        VersionBomOriginView versionBomOriginView = new VersionBomOriginView();
+        projectVersionComponentVersionView.setOrigins(List.of(versionBomOriginView));
```

Comment:
Could just do `projectVersionComponentVersionView.setOrigins(List.of(new VersionBomOriginView()));` if versionBomOriginView isnt used again

Reply:
This will be fixed in a followup PR containing test improvements for this section.